documentation/Using-the-SDK/Building-a-table.md

```diff
@@ -1,11 +1,11 @@
 # Building a Table
 
-Table construction happens by interacting with an instance of an `ITableBuilder`. Broadly, `Tables` consist of 3 parts: columns, which are composed of `ColumnConfigurations` and `Projections`, and `TableConfigurations`. An `ITableBuilder` has methods that accept all three of these objects.
+Table construction happens by interacting with an instance of an `ITableBuilder`. Broadly, `Tables` consist of `TableConfigurations` and Columns. Columns can be further broken down into `ColumnConfigurations` and `Projections`. An `ITableBuilder` has methods to accept Columns and `TableConfigurations`.
 
 * [Column](#column)
   * [ColumnConfiguration](#columnconfiguration)
   * [Projection](#projection)
-* [Combining ColumnConfiguration and Projections](#combining-columnconfiguration-and-projections)
+  * [Combining ColumnConfiguration and Projections](#combining-columnconfiguration-and-projections)
```

Comment:
The indent level is supposed to match the heading level. Can you update
`## Combining ColumnConfiguration and Projections` to `### Combining ColumnConfiguration and Projections`?